superset/assets/src/explore/visTypes.jsx

```diff
@@ -155,11 +155,16 @@ export const visTypes = {
         controlSetRows: [
           ['pie_label_type'],
           ['donut', 'show_legend'],
-          ['labels_outside'],
+          ['show_labels', 'labels_outside'],
           ['color_scheme'],
         ],
       },
     ],
+    controlOverrides: {
+      row_limit: {
+        default: 25,
```

Comment:
what do you think about making the default even lower, like 15?

Reply:
25 is quite large but I think we should push for the change in open source that way it ensures that future upstream changes don't override our configuration.